Tasks/Blashko/Task3/Task3_3.py

```diff
@@ -0,0 +1,25 @@
+"""3. Write a program that takes a string as input and returns a dictionary with the count of each word in the string.
+"""
+
+# получаем ввод от пользователя
+string = input('Enter string:\n')
+
+# задаем список символов, которые хотим удалить из строки
+symb = ['.', ',', ':', ';', '?', '!', '-', '_', '(', ')', '@', '"', "'"] 
+
+# избавляемся от ненужных символов
+for i in string:
+    if i in symb:
+        string = string.replace(i, '')
```

Comment:
how about

for s in symb:
    string = string.replace(i, '')

I encourage you to rationalize, why it's better to go with loop by the symb list (btw it can be simplified into str) and why the if ... in ... is not required